### PR TITLE
[MISC] Remove 'using namespace seqan3' from test/unit/alignment/confi…

### LINKED ISSUE #1622: [MISC] Remove 'using namespace seqan3' from test/unit/alignment/configuration
Remove 'using namespace seqan3' from
- [x] test/unit/alignment/configuration

### PR DESCRIPTION
**Pull request by Irallia**
…guration.

Resolves #1622.

### CODE REVIEW

File: test/unit/alignment/configuration/align_config_edit_test.cpp [hunk 1]
```diff
@@ -9,34 +9,35 @@
...
 #include <seqan3/alignment/configuration/align_config_edit.hpp>
 
-using namespace seqan3;
-
 TEST(align_cfg_edit, is_global)
 {
-    EXPECT_EQ((std::is_same_v<std::remove_reference_t<decltype(get<align_cfg::mode>(align_cfg::edit).value)>,
-                              detail::global_alignment_type>),
+    using type_of_cfg = decltype(seqan3::get<seqan3::align_cfg::mode>(seqan3::align_cfg::edit).value);
+    EXPECT_EQ((std::is_same_v<std::remove_reference_t<type_of_cfg>,
```
Comment (joergi-w) [inline test/unit/alignment/configuration/align_config_edit_test.cpp]: use `EXPECT_TRUE` instead ;)

File: test/unit/alignment/configuration/align_config_score_test.cpp [hunk 2]
```diff
@@ -13,67 +13,69 @@
...
 {
-    EXPECT_TRUE((std::is_default_constructible_v<detail::align_config_score<aminoacid_scoring_scheme<>>>));
+    EXPECT_TRUE((std::is_default_constructible_v<seqan3::detail::align_config_score<seqan3::aminoacid_scoring_scheme<>>>));
 }
 
 TEST(align_config_score, on_align_config)
 {
-    using config_score_type = detail::align_config_score<aminoacid_scoring_scheme<>>;
+    using config_score_type = seqan3::detail::align_config_score<seqan3::aminoacid_scoring_scheme<>>;
+    using typename_of_algin_config = typename seqan3::detail::on_align_config<seqan3::align_cfg::id::score;
```
Comment (joergi-w) [inline test/unit/alignment/configuration/align_config_score_test.cpp]:
```suggestion
    using typename_of_align_config = typename seqan3::detail::on_align_config<seqan3::align_cfg::id::score;
```

File: test/unit/alignment/configuration/align_config_score_test.cpp [hunk 3]
```diff
@@ -13,67 +13,69 @@
...
 
-    EXPECT_TRUE((std::is_same_v<typename detail::on_align_config<align_cfg::id::score>::invoke<config_score_type>,
+    EXPECT_TRUE((std::is_same_v<typename_of_algin_config>::invoke<config_score_type>,
                  std::true_type>));
-    EXPECT_TRUE((std::is_same_v<typename detail::on_align_config<align_cfg::id::score>::invoke<bar>,
+    EXPECT_TRUE((std::is_same_v<typename_of_algin_config>::invoke<bar>,
```
Comment (joergi-w) [inline test/unit/alignment/configuration/align_config_score_test.cpp]: typo: `align` (2x)

File: test/unit/alignment/configuration/align_config_edit_test.cpp [hunk 4]
```diff
@@ -12,8 +12,8 @@
 TEST(align_cfg_edit, is_global)
 {
     using type_of_cfg = decltype(seqan3::get<seqan3::align_cfg::mode>(seqan3::align_cfg::edit).value);
-    EXPECT_EQ((std::is_same_v<std::remove_reference_t<type_of_cfg>,
-                              seqan3::detail::global_alignment_type>),
+    EXPECT_TRUE((std::is_same_v<std::remove_reference_t<type_of_cfg>,
+                                seqan3::detail::global_alignment_type>),
                true);
```
Comment (joergi-w) [inline test/unit/alignment/configuration/align_config_edit_test.cpp]: You need to remove this `, true` when changing from `_EQ` to `_TRUE`. 
Sorry for not being more specific earlier.

Comment (Irallia) [reply test/unit/alignment/configuration/align_config_edit_test.cpp]: This happens when you stop thinking. I corrected the commit.

**Pull request merged**